### PR TITLE
Update and rename Duolingo.com.xml to Duolingo.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

**Comment by pipboy96** [comment]
I think there may be more subdomains, please use Amass to get them.

File: src/chrome/content/rules/Duolingo.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!--
+	Mismatched:
+		^duolingo.cn		equal to www.duolingo.cn
+	Timeout:
+		sponsorships.duolingo.com
+-->
+<ruleset name="Duolingo.com">
+	<target host="duolingo.com" />
+	<target host="*.duolingo.com" />
```
Comment (pipboy96) [inline src/chrome/content/rules/Duolingo.xml]: Please avoid <kbd>*</kbd> if it's even possible.

File: src/chrome/content/rules/Duolingo.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	<target host="englishtest.duolingo.cn" />
+
+	<securecookie host=".+" name=".+" />
+
+	<exclusion pattern="^http://sponsorships\.duolingo\.com/" />
+		<test url="http://sponsorships.duolingo.com/" />
+		<test url="http://www.duolingo.com/" />
+		<test url="http://api.duolingo.com/" />
+		<test url="http://englishtest.duolingo.com/" />
+		<test url="http://de.duolingo.com/" />
```
Comment (pipboy96) [inline src/chrome/content/rules/Duolingo.xml]: Can you do the opposite, e.g. only include working targets.

Comment (ivysrono) [reply src/chrome/content/rules/Duolingo.xml]: are there tools to find subdomains all?

Comment (pipboy96) [reply src/chrome/content/rules/Duolingo.xml]: @ivysrono https://github.com/OWASP/Amass

**Comment by pipboy96** [comment]
Did you test all subdomains of `.cn` too?

**Comment by ivysrono** [comment]
> 
> 
> Did you test all subdomains of `.cn` too?

I did not found more subdomains of .cn

**Pull request merged**